executor/batch_point_get.go

```diff
@@ -155,9 +169,47 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	}
 
 	// Fetch all values.
-	values, err1 := reader.BatchGet(ctx, keys)
-	if err1 != nil {
-		return err1
+	var values map[string][]byte
+	if e.lock {
```

Comment:
The `initialize` function seems a bit long, better  put lock into another function like point get `LockKeysIfNeeded`